lib/pwwka/logging.rb

```diff
@@ -18,11 +18,12 @@ def logger
     def logf(format,params)
       level = params.delete(:at) || Pwwka.configuration.log_level
       params[:payload] = params["payload"] if params["payload"]
-      params[:payload] = "[omitted]" if params[:payload] && LEVELS[Pwwka.configuration.payload_logging.to_sym] > LEVELS[level.to_sym]
+      if Pwwka.configuration.omit_payload_from_log?(level)
+      #if LEVELS[Pwwka.configuration.payload_logging.to_sym] > LEVELS[level.to_sym]
```

Comment:
do we want to do away with this line? or are we fine with leaving it as a comment?

Reply:
Oops, that was during refactoring.  Can you remove and re-push?  Thanks!!